artifactory_test.go

```diff
@@ -994,6 +994,17 @@ func TestArtifactoryUploadAndExplode(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+func TestUploadAndSyncDeleteCaseSensitivity(t *testing.T) {
+	initArtifactoryTest(t, "")
+	runRt(t, "upload", "--sync-deletes",tests.RtRepo1+"/", path.Join("testdata", "syncdeletes", "*"), tests.RtRepo1+"/")
```

Comment:
Would it be better to make sure files are being deleted by doing two separate uploads?